functions/src/notifications/sendEmail.ts

```
@@ -0,0 +1,41 @@
+import { NotificationParams } from '../types/Notification'
+import * as sgMail from '@sendgrid/mail'
+import { logEmailSent } from '../sumologic/sumologic'
+
+export const sendEmail = async (
+    params: NotificationParams,
+    messageBody: string,
+    subject: string
+) => {
+    if (!params.sendGridCredentials) {
+        return Promise.reject('Missing SendGrid env')
```

pls consider throwing an error here, instead of returning a `reject` promise